proto/toolchain.bzl

```diff
@@ -16,7 +16,7 @@
 
 def file_stem(f):
```

Comment:
This function should remain a helper that takes '/some/path/file.txt' and returns 'file', can you move the `replace` to L52?

`paths = ["%s/%s" % (output_dir, file_stem(i).replace("-", "_") for i in protos.to_list()]`

Reply:
Yes that makes sense. (I didn't notice the other usages.)